indico/modules/users/models/export.py

```diff
@@ -30,17 +32,17 @@ class DataExportRequestState(RichIntEnum):
 
 class DataExportOptions(RichStrEnum):
     __titles__ = {
-        'personal_data': L_('Personal data'),
```

Comment:
What's the reason for no longer using lazy strings here?

(even though I'm not sure if they were needed in the first place, since outside a request `_(...)` results in a lazy string anyway)

Reply:
yeah, that's the reason. Thought I'd update it while I'm at it, but I can leave those changes out

Reply:
no, it's fine, makes sense to only use `L_()` where it's necessary